tests/test_in_progress_sla_trending_report/test_in_progress_sla_trending_raw_data_table.py

```diff
@@ -0,0 +1,277 @@
+import pytest
+import json
+import uuid
+from datetime import timedelta
+from time import sleep
+
+import jq
+
+from helpers.datetime_helper import (datetime_utc_now,
+                                     generate_report_end_date,
+                                     generate_report_start_date)
+from helpers.splunk import (create_sample_event,
+                            create_transfer_compatibility_payload,
+                            get_telemetry_from_splunk,
+                            set_variables_on_query, create_error_payload, create_ehr_response_payload)
+from tests.test_base import EventType, TestBase
+
+
+class TestInProgressSlaTrendingRawDataTable(TestBase):
+
+    @pytest.mark.parametrize("column_indicator", ["yesterday", "two_days_ago"])
+    def test_in_progress_sla_trending_raw_data_table_select_column_token(self, column_indicator):
+        # Arrange
+        index_name, index = self.create_index()
+
+        if column_indicator == "yesterday":
+            td_hours = 25
+            num_events = 1
+        elif column_indicator == "two_days_ago":
+            td_hours = 49
+            num_events = 2
+        else:
+            raise ValueError('column value should be "yesterday" or "two_days_ago"')
```

Comment:
Do you know if we need to raise an error? Since we set which value we're testing per run?

Reply:
this is to ensure that someone doesn't change it by accident.